multigeiger/speaker.cpp

```diff
@@ -160,6 +169,12 @@ void tick_enable(bool enable) {
   }
 }
 
+void alarm() {
+  portENTER_CRITICAL_ISR(&mux_audio);
+  isr_audio_sequence = alarm_sequence;
+  portEXIT_CRITICAL_ISR(&mux_audio);
+}
+
```

Comment:
this is not called from an ISR, so you have to use the calls without _ISR.

Reply:
my mistake, removed.